index.js

```diff
@@ -247,10 +231,9 @@ app.post('/close-window', async (req, res) => {
         verbose('simpleFundsTransfer', simpleFundsTransfer);
         verbose('opSettlementsEp', config.opSettlementsEp);
         const sharedLib = new Model({ endpoint: config.opSettlementsEp });
-        await sharedLib.postSettlementFile(settlement.id, matrix,
+        await sharedLib.postSettlementFile(settlement.id, '',
```

Comment:
Is there any reason we keep this around?

Reply:
We'll need to have a placeholder since the casablanca lib is expecting a matrix value. This will be cleaned up in another moment.

Reply:
We'll need to have a placeholder since the finance-portal-lib is expecting a matrix value. This will be cleaned up in another moment.

Reply:
You mean we need to store the third argument to be used later on?

Reply:
Yes, since that value will be downloaded later as the `payment matrix` in the portal. but it's not working and it will be removed later on. So it's a temporal fix until we have to remove everything related to the payment matrix.